frontend/pages/beta/model/new/upload.tsx

```diff
@@ -0,0 +1,223 @@
+          seldonVersion: { 'ui:widget': 'seldonVersionSelector' },
+        },
+      },
+      [],
+      defaultState
+    )
+
+    steps.push(
+      createStep({
+        schema: {
+          title: 'Files',
+        },
+        state: {
+          binary: undefined,
+          code: undefined,
+        },
+        schemaRef: reference,
+
+        type: 'Data',
+        index: steps.length,
+        section: 'files',
+
+        render: RenderFileTab,
+        renderBasic: RenderBasicFileTab,
+        isComplete: fileTabComplete,
+      })
+    )
+
+    steps.push(
+      createStep({
+        schema: {
+          title: 'Submission',
+        },
+        state: {},
+        schemaRef: reference,
+
+        type: 'Message',
+        index: steps.length,
+        section: 'submission',
+
+        render: () => null,
+        renderButtons: renderSubmissionTab,
+        isComplete: () => true,
+      })
+    )
+
+    for (const step of steps) {
+      step.steps = steps
+    }
+
+    setSplitSchema({ reference, steps })
+  }, [currentSchema, user])
+
+  const errorWrapper = MultipleErrorWrapper(
+    `Unable to load upload page`,
+    {
+      isDefaultSchemaError,
+      isSchemasError,
+      isCurrentUserError,
+    },
+    MinimalErrorWrapper
+  )
+  if (errorWrapper) return errorWrapper
+
+  if (isDefaultSchemaLoading || isSchemasLoading || isCurrentUserLoading) {
+    return null
+  }
+  const Loading = <Wrapper title='Loading...' page='deployment' />
+
+  if (isDefaultSchemaLoading || !defaultSchema) return Loading
+  if (isSchemasLoading || !schemas) return Loading
+  if (isCurrentUserLoading || !currentUser) return Loading
+
+  const onSubmit = async () => {
+    setError(undefined)
+
+    if (!splitSchema.steps.every((e) => e.isComplete(e))) {
+      return setError('Ensure all steps are complete before submitting')
+    }
+
+    const data = getStepsData(splitSchema, true)
+    const form = new FormData()
+
+    data.schemaRef = currentSchema?.reference
+
+    form.append('code', data.files.code)
+    form.append('binary', data.files.binary)
+    form.append('docker', data.files.docker)
+    delete data.files
+
+    form.append('metadata', JSON.stringify(data))
+    setModelUploading(true)
+
+    await axios({
+      method: 'post',
+      url: '/api/v1/model',
+      headers: { 'Content-Type': 'multipart/form-data' },
+      data: form,
+      onUploadProgress: (progressEvent) => {
+        setUploadPercentage(progressEvent.total ? (progressEvent.loaded * 100) / progressEvent.total : 0)
+      },
+    })
+      .then((res) => router.push(`/model/${res.data.uuid}`))
+      .catch((e) => {
+        setModelUploading(false)
+        setError(e.response.data.error.message)
+        return null
+      })
+    return null
+  }
+
+  return (
+    <Paper variant='outlined' sx={{ my: { xs: 3, md: 6 }, p: { xs: 2, md: 3 } }}>
+      <Grid container justifyContent='space-between' alignItems='center'>
+        <Box />
+        <SchemaSelector
```

Comment:
The beta pages will handle the schema selection on another page, so I'd recommend removing it. We should be able to remove all of the other schema related stuff from the page as well (such as the change/setting new schema functions). 

For now we can just fetch the minimal schema and use that to render the form for test purposes.

Reply:
Again, removed in BAI-742